sql/sql_select.cc

```diff
@@ -14928,16 +14928,16 @@ bool Item_func_eq::check_equality(THD *thd, COND_EQUAL *cond_equal,
     The function performs the substitution in a recursive descent by
     the condtion tree, passing to the next AND level a chain of multiple
     equality predicates which have been built at the upper levels.
-    The Item_equal items built at the level are attached to other 
+    The Item_equal items built at the level are attached to other
     non-equality conjucts as a sublist. The pointer to the inherited
     multiple equalities is saved in the and condition object (Item_cond_and).
-    This chain allows us for any field reference occurence easyly to find a 
-    multiple equality that must be held for this occurence.
+    This chain allows us for any field reference occurrence easyly to find a
```

Comment:
occurrence easily

Reply:
@an3l please adjust the commit to also fix the "easily" word, then ok to push.

Reply:
@ottok do you want to fix this please, because of github statistics (merged over closed)?

Reply:
Done.